service/src/main/java/bio/terra/workspace/amalgam/tps/TpsApiDispatch.java

```diff
@@ -114,27 +117,24 @@ private static PolicyInput policyInputFromApi(ApiTpsPolicyInput apiInput) {
       throw new TpsInvalidInputException("PolicyInput namespace and name cannot be null");
     }
 
-    Map<String, String> data;
-    if (apiInput.getAdditionalData() == null) {
-      // Ensure we always have a map, even if it is empty.
-      data = new HashMap<>();
-    } else {
-      data =
-          apiInput.getAdditionalData().stream()
-              .collect(Collectors.toMap(ApiTpsPolicyPair::getKey, ApiTpsPolicyPair::getValue));
+    Multimap<String, String> data = ArrayListMultimap.create();
```

Comment:
is it valid to have multiple `additionalData` fields with the same key?

Reply:
yes, it switched to a multimap.